lib/zinnia.js

```diff
@@ -48,6 +54,7 @@ const updateAllSourceFiles = async () => {
   const modules = await Promise.all(
     Object
       .values(ZINNIA_MODULES)
+      .filter(({ module }) => module.includes(MODULE_FILTER))
```

Comment:
I would expect `MODULE_FILTER` to support more than one module, e.g. `MODULE_FILTER=spark,bacalhau`.

I would also expect this filter to be applied to non-Zinnia modules like Bacalhau.

At the same time, I think this is all relatively low priority. I propose to either
- Change the env var name to better convey the intent (e.g. `SINGLE_ZINNA_MODULE`)
- Drop module filtering completely and wait until there is user demand

Reply:
> If you work on the spark or voyager integration, I'm sure you'll find use for it too.

I wasn't aware this is something you found useful yourself. I agree to keep this feature.

> As we only have two modules, filtering multiple via `a,b` isn't required. I was planning to add this once we have a 3rd module. To me nothing about `MODULE_FILTER` communicates that I can pass a comma separated list. Maybe we can rename it to `MODULE_GREP`?

I see. I would expect MODULE_GREP to support regular expressions. MODULE_FILTER is better in this aspect.

My main concern is about making this feature easy to understand and avoid confusion.

I don't have any more suggestions. The current implementation seems to be the best we can think of now. 🤷🏻 

> The Bacalhau module is an experimental one and not being updated (and not being used). I'm rather leaning towards removing it instead.

+1 to remove it so that we can simplify our work by not having to consider Bacalhau for consistency across all modules.

Reply:
> My main concern is about making this feature easy to understand and avoid confusion.
> I don't have any more suggestions. The current implementation seems to be the best we can think of now. 🤷🏻

Ok let's ship it like this. We can always iterate on it. I don't want to block the Voyager integration on this simple env var.

> > The Bacalhau module is an experimental one and not being updated (and not being used). I'm rather leaning towards removing it instead.
> 
> +1 to remove it so that we can simplify our work by not having to consider Bacalhau for consistency across all modules.

Cool! https://github.com/filecoin-station/core/pull/326